proposed/2023/Add-NoHttpCacheOption.md

```diff
@@ -0,0 +1,62 @@
+
+To make things clearer, a new CLI option has been introduced: `--no-http-cache`. When you use this flag, it explicitly disables only the HTTP cache, leaving the global packages folder untouched. The previous options are still there; And they will have warnings to help move people towards the better named options. In addition, we will be hiding the existing `--no-cache` option from the CLI help output. The `--no-htt-cache` option will be shown in the CLI help output. This will help guide users to the new option.
+
+Here's a simple restore example:
+
+Old way (still works): `dotnet restore --no-cache`
+New way (clearer): `dotnet restore --no-http-cache`
+### Technical explanation
+
+<!-- Explain the proposal in sufficient detail with implementation details, interaction models, and clarification of corner cases. -->
+The proposal is to add a new option `-NoHttpCache` which will set the exact same `NoCache` option boolean. For now if either of these options are set, `NoCache` will be set to true.
+## Drawbacks
+
+<!-- Why should we not do this? -->
+
+## Rationale and alternatives
+
+<!-- Why is this the best design compared to other designs? -->
+<!-- What other designs have been considered and why weren't they chosen? -->
+<!-- What is the impact of not doing this? -->
+This design is the best because it introduces explicitness without breaking existing workflows; the older flags are not deprecated and still work as before. Alternative designs like modifying the behavior of existing flags or deprecating them were considered but ruled out because we do not want to confuse our users. 
+
+## Prior Art
+
+<!-- What prior art, both good and bad are related to this proposal? -->
+<!-- Do other features exist in other ecosystems and what experience have their community had? -->
+<!-- What lessons from other communities can we learn from? -->
+<!-- Are there any resources that are relevant to this proposal? -->
+* None
+## Unresolved Questions
+
+<!-- What parts of the proposal do you expect to resolve before this gets accepted? -->
+<!-- What parts of the proposal need to be resolved before the proposal is stabilized? -->
+<!-- What related issues would you consider out of scope for this proposal but can be addressed in the future? -->
+To ensure a smooth transition, the current `-NoCache` options will be preserved. However, we would also like our users to know this option will only stop Http caching and that there is a new option `-NoHttpCache` which is clearer and does the same thing. As a result, printing a waring when users use the `-NoCache` option is ideal. I am not sure if the following is a sufficient warning \
+&ensp; `"NoCache is deprecated in favor of the appropriately named NoHttpCache."` 
```

Comment:
Hey @JonDouglas @albarry4, what do you think of this message?  Please let me know if you have any suggestion.